lib/classes/CLI.js

```diff
@@ -321,8 +321,18 @@ class CLI {
     process.stdout.write(chalk.yellow('.'));
   }
 
-  log(message, entity) {
-    this.consoleLog(`${entity || 'Serverless'}: ${chalk.yellow(`${message}`)}`);
+  log(message, entity, opts) {
+    const underline = opts ? opts.underline : false;
```

Comment:
If you make these in the form of `(opts && opts.underline !== undefined) ? opts.underline :false` it'll support incomplete `opts` objects